### DISCUSSION TITLE
Jak w Adguardzie na tel zgłosić nie usunięte reklamy?

### DISCUSSION THREAD

**Discussion by gzenio22** [MajkiIT/polish-ads-filter#22012] [Q&A]
Witam tym razem to ja zadam pytanie bo zbaraniałem mimo ze od lat korzystam już z aplikacji Adguarda mam mały kłopot a mianowicie w jaki sposob z poziomu telefonu wysłać zgłoszenie o niezablokowanych reklamach na jakiejs stronie?

Na PC jest to proste klikasz w ikone adguarda i zgłoś tę strone albo problem (W zaleznosci czy to adguard PC czy wtyczka do przeglądarki) Ale w apce na tel nie widze nigdzie opcji takiego przekierowania do formulaża a obstawiam ze gdzies jest a ja jestem poprostu slepy

**Comment by garry-ut99** [comment]
- https://adguard.com/kb/guides/report-website/#android (po angielsku)
- https://adguard-com.translate.goog/kb/guides/report-website/?_x_tr_sl=en&_x_tr_tl=pl&_x_tr_hl=pl&_x_tr_pto=wapp#android (przetłumaczone na polski)

**Comment by krystian3w** [comment]
Albo niezalecaną metodą przez formularz na github: https://github.com/AdguardTeam/AdguardFilters/issues/new/choose

Pewnie i tak nie odnajdziesz się, jak znajdziesz błąd w działaniu programu gdzie go dokładnie umieścić:
- ExtendedCSS
- CoreLibs
- FiltersCompiler
- tsurlfilter
